src/components/Home/Home.js

```diff
@@ -2,10 +2,30 @@ import React, { Component } from 'react';
 import CompoCocktail from './CompoCocktail';
 import Quizz from './Quizz';
 import video from '../Images/Strawberries.mp4';
-import cuba from '../Images/cuba-libre.jpg';
 import './Home.css';
+import axios from 'axios';
 
 class Home extends Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      drinks: []
+    };
+  }
+
+  componentDidMount () {
+    axios
+      .get('https://www.thecocktaildb.com/api/json/v1/1/random.php')
+      .then((response) => {
+        this.setState({ drinks: response.data.drinks });
```

Comment:
Je ne comprends pas pourquoi vous sockez votre cocktail du jour sous forme de tableau (Object.values) et pourquoi vous faites plusieurs setState.  
Conseil : dans votre state, mettez une clé "randomCocktail" que vous initialisez à null. Quand l'api répond, vous pouvez faire un this.setState({randomCocktail: response.data.drinks[0]}). Ensuite, il suffira d'accéder aux infos du cocktail en lui donnant des nom de propriété, pas des index dans un tableau :p 
Pensez également à gérer le fait que le cocktail n'aura pas encore été chargé au début, si tel n'est pas le cas affichez "cocktail du jour en cours de chargement..." (ou quelque chose comme ça) à la place des infos sur le cocktail.